ev3dev/core.py

```diff
@@ -2623,6 +2638,29 @@ def rate_and_angle(self):
         self.mode = self.MODE_GYRO_G_A
         return self.value(0), self.value(1)
 
+    @property
+    def tilt_angle(self):
+        self.mode = self.MODE_TILT_ANG
+        return self.value(0)
+
+    @property
+    def tilt_rate(self):
+        self.mode = self.MODE_TILT_RATE
+        return self.value(0)
+
+    @property
+    def direct(self):
```

Comment:
Docs for `direct`?

Reply:
Should probably be `_direct`. I don't expect that it would be used normally, if at all.